_data/projects/facedb.yml

```
@@ -0,0 +1,12 @@
+name: FaceDB
+desc: A face database for large-scale face recognition
+site: https://github.com/shhossain/facedb
+tags:
+- python
+- machine-learning
+- deep-learning
+- face-recognition
+- vector-database
+upforgrabs:
+  name: help-wanted
```

The label name has a space in it

```suggestion
  name: help wanted
```